tests/lib/rules/no-unused-prop-types.js

```diff
@@ -3136,6 +3135,73 @@ ruleTester.run('no-unused-prop-types', rule, {
 
         Foo.defaultProps = Object.assign({});
       `
+    }, {
+      code: `
+        const Foo = (props) => {
+          const { foo } = props as unknown;
+          (props as unknown).bar as unknown;
```

Comment:
could we also add this same component as a failing test, with only one of these two propTypes present?